hawk/app/assets/stylesheets/shared/_misc.scss

```diff
@@ -32,3 +32,30 @@
     border-radius: 20px;
   }
 }
+
+.diff_sub {
+  background: #fdd;
+}
+
+.diff_add {
+  background: #dfd;
+}
+
+.diff_chg {
+  background: #ddf;
+}
+
+
+td.diff_header {
+color: #fff;
+}
+
+table.diff {
+ width: 100%;
+ font-family:monospace;
+ border: none;
+}
+
+.diff_next {
+  background-color: #c0c0c0;
```

Comment:
Color literals like `#c0c0c0` should only be used in variable declarations; they should be referred to via variable everywhere else.